website/docs/core-concepts/stacks/yaml-functions/include.mdx

```diff
@@ -75,6 +98,35 @@ components:
 ```
 </File>
 
+<File title="stack.yaml">
+```yaml
+import: !include config/import.yaml
```

Comment:
Somewhere we need to explain the search paths to find `config/import.yaml`. What is it relative to? e.g. the component, the current stack config file, the base path, etc?

Reply:
Let's move the part about how it loads files into a chapter titled `## Import Search Paths`. 

Let's rename "Supported File Protocols" to "Supported File Formats" and make it only about the types of files it handles. 

Also, I believe it doesn't respect file extensions - we should list that. It guesses the type of format based on the file's contents. Also, can we support a raw format? IMO, if the file ends in `.txt` this is how it should be handled.

Lastly, we should have a chapter on `## Remote Includes` and how that works and mention token injection.